laravel/app/Services/AutoDeliveryService.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace App\Services;
+
+use App\Models\OrderItem;
+use App\Models\Inventory;
+use App\Services\GoogleSheet;
+use App\Models\TransportCompany;
+use App\Models\User;
+use Carbon\Carbon;
+use Illuminate\Support\Facades\DB;
+use \Exception;
+use App\Mail\AutoDeliverySystemNotification;
+use Illuminate\Support\Facades\Mail;
+
+class AutoDeliveryService
+{
+    public static function AutoDeliveryNoOrder($ship_date)
+    {
+        $users = User::all();
+        $users_mail_lists = $users->pluck('email')->toArray();
+
+        $transports = TransportCompany::all();
+        $transport_mail_lists = $transports->pluck('email')->toArray();
+              
+        $mail_lists = array_merge($users_mail_lists, $transport_mail_lists);
+        $mail_text = '納入日'.$ship_date.'この日の新しい注文はございません。
+        　※別途送付済みの指示書がある場合は、そちらを正としてご手配を進めて下さい。';
+        Mail::to($mail_lists)->send( new AutoDeliverySystemNotification($mail_text) );
+    }
+
+    public static function AutoDeliveryExecute($ship_date, $order_indexes)
```

Comment:
Autoはとってしまって良さそうです。
また少しメソッドが長いので、今後また時間が空いた際にメソッド分割を考えて行きましょう
```suggestion
    public static function DeliveryExecute($ship_date, $order_indexes)
```